crates/uv/src/commands/project/run.rs

```diff
@@ -108,6 +111,24 @@ pub(crate) async fn run(
     } else {
         debug!("Syncing ephemeral environment.");
 
+        // Extract the project settings.
+        let InstallerSettings {
+            index_locations: _,
+            index_strategy: _,
+            keyring_provider,
+            resolution: _,
+            prerelease: _,
+            config_setting: _,
+            exclude_newer: _,
+            link_mode: _,
+            compile_bytecode: _,
```

Comment:
Are we going to use some of these in the future?

Reply:
No, but I think we should be forced to think about it if we add new settings.

Reply:
I feel like a lot of these actually apply though? Like don't we care about the index locations and strategy here?

Reply:
Where? This is just for `Toolchain::find_or_fetch`.

Reply:
We could omit `keyring_provider` entirely if you want, since we know the URLs that will be fetched here.

Reply:
Oh sorry I didn't realize that was the context. I wouldn't use the keyring for `find_or_fetch`.

Reply:
Thanks for clarifying.

Reply:
Kk will remove!